src/components/RequestRepository/index.tsx

```diff
@@ -53,6 +53,46 @@ const RequestRepository = () => {
         Header: t('requestRepository.table.requestType'),
         accessor: 'requestType'
       },
+      {
+        Header: t('intake:fields.grtDate'),
+        accessor: 'grtDate',
+        Cell: ({ row, value }: any) => {
+          if (value) {
+            return DateTime.fromISO(value).toLocaleString(DateTime.DATE_FULL);
+          }
+
+          // If date is null, return button that takes user to page to add date
+          return (
+            <UswdsLink
+              data-testid="add-grt-date-cta"
+              asCustom={Link}
+              to={`/governance-review-team/${row.original.id}:id/dates`}
```

Comment:
I think this URL is incorrect.

Reply:
You are right, i accidentally left the :id string in there. I will change that now. 

Weirdly though, it works in my local env (I just checked again). 

http://localhost:3000/governance-review-team/c36f957a-6b7a-4665-9e3a-19360414053f:id/dates

^ That URL takes me to the correct page even though it has a ":id" appended onto the system id.

Reply:
So yeah you can basically do whatever you want to the system id in the URL and still get to the page

http://localhost:3000/governance-review-team/Random-stuff-e551e77b-b12a-4casdfasdfa-Testing-sdfa90-a7a9-f001bed9casdf9ec3-asdNick-is-cool-fasdfasdfasdfasdfa/dates

^ that still takes you to the page, i'm assuming the date change wont actually take effect if the URL is messed up